server/command/install.go

```diff
@@ -16,9 +16,11 @@ import (
 func (s *service) executeInstall(params *params) (*model.CommandResponse, error) {
 	appID := ""
 	appSecret := ""
+	appWebhookSecret := ""
 	fs := pflag.NewFlagSet("", pflag.ContinueOnError)
 	fs.StringVar(&appID, "app-id", "", "ID of the app")
 	fs.StringVar(&appSecret, "app-secret", "", "App secret")
+	fs.StringVar(&appWebhookSecret, "app-webhook-secret", "", "App webhook secret")
```

Comment:
Should the webhook secret be autogenerated from mattermost? If I am not mistaken, expanding the app will send also the webhook secret with the call, so if the app needs it, it can take it from there.

Reply:
I can generate the secret automatically. @levb and I discussed this also.  I like the idea, but we scoped the PR as this simple implementation where the secret is provided by the installer.  

If autogenerated, that would be great.  We would need to provide a method for retrieving the webhook secret (or better yet the link for the endpoint).  Maybe the zendesk DM after install?  

What if the user wants to change the webhook secret? I think that is only available by doing another `install`

Reply:
Ideally, I'd like to provide a `/zendesk setup-webhook` command to go and create the webhook without having to go into `zendesk` itself and manually create

Reply:
What is a use case for setting up  a new webhook secret? Would a slash command like `/zendesk setup-webhook --force` to re-setup the webhook be fine?

Reply:
Should we fore now generate the secret and output it after the install process for the admin?

Reply:
The reason in general for regenerating secrets is that it has been compromised, and has to be reset for security reasons. (Or security policies to reset every X time). I think it is a needed feature down the line, but 0/5 on how urgent it is.

Reply:
1/5 keep it manually entered for now, maybe pre-initialize on install (use App.Secret?). Then have a special command to change it.

We can always migrate common admin commands to the framework over time, so `/zendesk webhook secret` may eventually become `/apps webhook secret --app zendesk`. Anyway, simpler is better for now.

Reply:
The secret is generated on install now